include/session.php

```diff
@@ -40,12 +40,12 @@ function ref_session_write ($id,$data) {
 	if($session_exists) {
 		$r = q("UPDATE `session`
 				SET `data` = '%s'
-				WHERE `data` != '%s' AND `sid` = '%s'",
+				WHERE `sid` = '%s' AND `data` != '%s'",
 				dbesc($data), dbesc($data), dbesc($id));
```

Comment:
Never mind, the condition wasn't the issue, all the updates on my server take at least 40ms. Reverting to original, although I couldn't help myself formatting and documenting the function.